src/App.tsx

```diff
@@ -20,15 +72,13 @@ export const App: React.FC = () => {
               <TodoFilter />
             </div>
 
-            <div className="block">
-              <Loader />
-              <TodoList />
-            </div>
+            <div className="block">{loading ? <Loader /> : <TodoList />}</div>
+
           </div>
```

Comment:
```suggestion
            <div className="block">
              {loading 
                ? <Loader /> 
                : <TodoList />
              }
            </div>
          </div>
```
Fix formatting to make your code easier to read. Also, don't add empty lines between parent and child elements